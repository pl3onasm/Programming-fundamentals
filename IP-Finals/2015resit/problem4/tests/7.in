56
6 5 48 53 9 2 15 54 33 29 3 12 40 11 38 30 51 4 25 18 0 50 36 14 21 1 31 22 7 39 37 45 28 24 17 42 47 23 34 43 32 46 49 26 20 13 35 52 44 27 19 10 8 55 41 16
