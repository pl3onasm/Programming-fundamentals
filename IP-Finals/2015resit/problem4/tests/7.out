0 50
3 6
